package.json

```diff
@@ -72,5 +72,8 @@
     "lint-staged": "^13.1.0",
     "semantic-release": "^20.0.2",
     "typescript": "^4.9.5"
+  },
+  "dependencies": {
+    "jaro-winkler": "^0.2.8"
   }
```

Comment:
Add dependencies to the respective package, please. The root `package.json` in theory should not have any dependencies.

`npm i <dep> -w packages/<pkg-name>`